ExilenceNextApp/src/components/item-table/ItemTable.tsx

```diff
@@ -139,6 +142,7 @@ const ItemTable: React.FC<ItemTableProps> = ({
 
   const handleChangePage = (event: unknown, newPage: number) => {
     changePage(newPage);
+    pageIndex = 1;
```

Comment:
I think this is misplaced?

Isn't this controlled through state in the parent? Does setting this here even affect anything?